### ISSUE TITLE
Orbitcontrols removes mouseup event from all child elements

### ISSUE THREAD

**Issue by TjeerdvanDijk** [mrdoob/three.js#24919]
**Describe the bug**

Having a child element inside of the orbitcontrol's domElement will prevent the child element from receiving mouseUp/pointerUp events.

**To Reproduce**
1.Create a div
2.Put canvas in div (your renderer canvas)
2.Put another element in div (I tried it with another canvas)
3.Instantiate threejs.
4.Instantiate orbitcontrols with the DIV as domElement.

now try to listen for mouseup events on the div's second element, it won't work

**What is causing this?**

I figured out that orbitcontrols is deleting the event. This happens in the function [``removePointer``](https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/OrbitControls.js#L984), to be specific when calling ``pointers.splice( i, 1 );``. You can comment it out and the event will work again.

**Expected behavior**

The mouseup and pointerup event works in the child element, and orbitcontrols does not alter the behavior.

**Platform:**

 - OS: Windows 11
 - Browser: Chrome
 - Three.js version: r146

**Comment by Mugen87** [comment]
Not deleting the event from the internal array will break touch interaction. 

It's best to share a live example first to demonstrate the issue. This makes it easier to evaluate the issue and think of possible workaround/solutions.

**Comment by Mugen87** [comment]
BTW: Your use case is a bit special since the `domElement` of `OrbitControls` is normally the renderer's canvas. And it's not possible to render child DOM elements of a canvas since they are intended as fallbacks (if the canvas is not supported).

Consider to use this setup: https://jsfiddle.net/9dpk1rtm/

**Comment by TjeerdvanDijk** [comment]
Hello, thank you for your example, how you did it is kind of how I solved it. I was trying to (visually) have a canvas on top of another, both would still be able to receive mouse/touch events (even at the area where they are stacked). The location of my canvas is not in the top corner of the screen, so that made it a bit trickier.

![image](https://user-images.githubusercontent.com/44869554/200376682-d4c0eef9-359b-476c-9979-365e4585fa16.png)

I moved the domElement back to the renderer canvas. I solved my use case by using the mouse events of the window, and used offsetLeft/offsetTop of the cube-canvas in combination with e.clientX e.clientY to determine where I clicked in the cube-canvas. 

The workaround works, but I don't know if there is a better way to do it
